packages/camera/camera_platform_interface/lib/src/types/camera_exception.dart

```diff
@@ -0,0 +1,20 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// This is thrown when the plugin reports an error.
+class CameraException implements Exception {
+  /// Creates a new camera exception with the given error code and description.
+  CameraException(this.code, this.description);
+
+  /// Error code.
+  // TODO(bparrishMines): Document possible error codes.
+  // https://github.com/flutter/flutter/issues/69298
+  String code;
+
+  /// Textual description of the error.
+  String description;
+
+  @override
+  String toString() => '$runtimeType($code, $description)';
```

Comment:
Can we not use `$runtimeType` here, and instead hardcode `CameraException`? When the web is running in release mode, the runtimeType will be minified IIRC.

Reply:
Good to know, it is updated.